next.config.mjs

```diff
@@ -1,4 +1,9 @@
 import { createVanillaExtractPlugin } from "@vanilla-extract/next-plugin";
+import createJITI from "jiti";
+import { fileURLToPath } from "url";
+
+const jiti = createJITI(fileURLToPath(import.meta.url));
+jiti("./src/env");
```

Comment:
[T3 Env: Validate Schema on Build](https://env.t3.gg/docs/nextjs#validate-schema-on-build-(recommended))

> We recommend you importing your newly created file in your `next.config.js`. This will make sure your environment variables are **validated at build time** which will save you a lot of time and headaches down the road. You can use [unjs/jiti](https://github.com/unjs/jiti) to import TypeScript files in your `next.config.js`.